.github/release/standard/Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM python:3.11-slim-bullseye
 ENV DEBIAN_FRONTEND noninteractive
 ENV TERM linux
 
+RUN apt-get update && apt-get --yes --no-install-recommends install build-essential
```

Comment:
This will increase the size of the container image significantly. Is there some package which does not provide binary wheels on PyPI?